### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by guanqun**

### CODE REVIEW

File: chalk-macros/src/lib.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ thread_local! {
 
 // When CHALK_DEBUG is enabled, we only allow this many frames of
 // nested processing, at which point we assume something has gone
-// awry.
```
Comment (shepmaster) [inline chalk-macros/src/lib.rs]: For what it's worth, "awry" is a valid English word, but it's probably worth changing it to something more understandable if it is causing confusion.

Comment (guanqun) [reply chalk-macros/src/lib.rs]: need to have a dictionary at my hand.

Comment (guanqun) [reply chalk-macros/src/lib.rs]: thanks for the info.

Comment (bjorn3) [reply chalk-macros/src/lib.rs]: away is not the right word.

**Pull request merged**